src/components/tooltip/Tooltip.tsx

```diff
@@ -325,12 +340,18 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                <div role="tooltip" aria-live="polite">
+                <div role="tooltip" aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
                     {tooltipInner}
                 </div>
             </div>
         ) : (
-            <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
+            <div
+                className={classes}
+                id={this.tooltipID}
+                role="tooltip"
+                aria-live="polite"
+                aria-hidden={isLabelMatchingTooltipText}
```

Comment:
this is to avoid repetition of texts being announced so if BOTH are the same means that `isLabelMatchingTooltipText` will be true and so aria-label and tooltipText doesn't have to be read with ONLY one of them being read is enough